### PR TITLE
fix: HTML tags should be removed from data coming from backend

### PR DESCRIPTION
**Pull request by pranavpandey1998official**
added stripHtml function to Strings in StringUtils.kt

added stripHtml function to Strings in StringUtils.kt and called it for event.detail and event.organizerDetail

fix: #1035 

whatsapp image 2019-02-06 at 3 34 12 am
whatsapp image 2019-02-06 at 3 34 08 am

### CODE REVIEW

**Comment by pranavpandey1998official** [comment]
@iamareebjamal project is using Retrofit2 which directly parsed json to objects, so other solution i could think is making a custom Deserializer for event data object . or we can attack the problem at view layer of app

**Comment by iamareebjamal** [comment]
No

**Comment by iamareebjamal** [comment]
You are suggesting chemotherapy for common cold. Just call the method in content getter. Why would you want entire custom serializer and parsing every field of every object to handle one case of one object?

**Comment by pranavpandey1998official** [comment]
@iamareebjamal I have made a custom constructor for event. and stiped the html their as I think we should save the response in the correct form. waiting for your thoughts

**Comment by iamareebjamal** [comment]
Why are you so eager to change all the fields when only change is required in the description field?

File: app/src/main/java/org/fossasia/openevent/general/event/Event.kt [hunk 1]
```diff
@@ -10,51 +10,50 @@ import com.github.jasminb.jsonapi.annotations.Id
 import com.github.jasminb.jsonapi.annotations.Relationship
 import com.github.jasminb.jsonapi.annotations.Type
 import org.fossasia.openevent.general.event.topic.EventTopic
+import org.fossasia.openevent.general.utils.stripHtml
 
 @Type("event")
 @JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
 @Entity
-data class Event(
+class Event(
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/event/Event.kt]: There's a reason why this is a data class, and it should remain so

**Comment by iamareebjamal** [comment]
I'm ignoring this PR until there is only one field change. This is tiring

**Comment by pranavpandey1998official** [comment]
organiser detail also show HTML tags and, I am changing all the fields because other fields can also show have HTML tags but if you are saying then I will only change these two fields, I have changed this to a normal class because I cannot make custom getters and setters for fields in primary constructor of data class.

**Comment by iamareebjamal** [comment]
Create a new getter in the data class `getStrippedDescription` etc

File: app/src/main/java/org/fossasia/openevent/general/event/Event.kt [hunk 2]
```diff
@@ -70,4 +71,12 @@ data class Event(
...
     var eventTopic: EventTopic? = null
-)
+) {
+    fun getStrippedDescription(): String? {
+        return this.description?.stripHtml()
+    }
+
+    fun getStrippedOrganizerDescription(): String? {
+        return this.organizerDescription?.stripHtml()
+    }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/event/Event.kt]: When you have created an extension function, then no need of these

Comment (pranavpandey1998official) [reply app/src/main/java/org/fossasia/openevent/general/event/Event.kt]: you said to make a new getter in data class how else I could have done it???

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/event/Event.kt]: Now that you have created an extension function, you can just write `event.description.getStrippedHtml()`

Comment (pranavpandey1998official) [reply app/src/main/java/org/fossasia/openevent/general/event/Event.kt]: k..

**Pull request merged**